uaclient/cli/tests/test_cli_attach.py

```diff
@@ -553,6 +560,7 @@ def test_attach_when_one_service_fails_to_enable(
         FakeConfig,
         event,
     ):
+        m_public_config.return_value = UserConfigData()
```

Comment:
`return_value` is already set above in the decorator, so no need to do it again here
```suggestion
```